scripts/end2end_examples.py

```diff
@@ -0,0 +1,119 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""
+To run this script you need to install papermill apart from argilla:
+
+$ pip install papermill
+
+Then run the script:
+
+$ python scripts/end2end_examples.py --help
+"""
+
+import os
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import Dict, Optional
+
+import papermill
+import typer
+
+EXAMPLES_FOLDER = (
+    Path(__file__).parent.parent / "docs/_source/tutorials_and_integrations/tutorials/feedback/end2end_examples"
+)
+OUTPUT_FOLDER = EXAMPLES_FOLDER / "output_notebooks"
+
+
+@dataclass
+class ExampleNotebook:
+    sort_index: int = field(init=False)
+    src_filename: Path
+    dst_filename: Path
+    parameters: Dict = field(default_factory=dict)
+
+    def __post_init__(self):
+        self.src_filename = EXAMPLES_FOLDER / self.src_filename
+        assert self.src_filename.exists(), f"File {self.src_filename} does not exist"
+        self.sort_index = int(self.src_filename.stem.split("-")[-1])
+        dst_folder = OUTPUT_FOLDER
+        if not dst_folder.exists():
+            dst_folder.mkdir(exist_ok=True)
+        self.dst_filename = dst_folder / self.dst_filename
+
+    def run(self):
+        try:
+            papermill.execute_notebook(
+                str(self.src_filename), str(self.dst_filename), parameters=self.parameters, kernel_name="python3"
+            )
+            print(f"✅  {self.src_filename.stem}")
```

Comment:
As a future improvement, to be sure the notebook does what it needs to do, we could have a step that executes a check_notebook_*.py file where all the assert/checks are done. But we can have a deep conversation on that without blocking this PR

Reply:
Sure, added issue to keep track of this: #4324